cmd/in_memory/dispatcher/main.go

```diff
@@ -25,11 +22,12 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"strings"
 	"time"
 
 	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
 	"github.com/knative/eventing/pkg/channelwatcher"
-	"github.com/knative/eventing/pkg/sidecar/swappable"
+	"github.com/knative/eventing/pkg/provisioners/swappable"
```

Comment:
why is this coming from provisioners?

Reply:
Receiver and dispatcher related common code is in provisioners. Swappable provides a wrapper around multichannelfanout handler.